google/cloud/spanner_dbapi/connection.py

```diff
@@ -165,6 +166,48 @@ def read_only(self, value):
             )
         self._read_only = value
 
+    @property
+    def staleness(self):
+        """Current read staleness option value of this `Connection`.
+
+        Returns:
+            dict: Staleness type and value.
+        """
+        return self._staleness
```

Comment:
In the interest of avoiding having to add `or {}` to a bunch of checks, perhaps it makes more sense to do one of the following:

1) Keep the default for `self._staleness` as `None` and update this line to `return self._staleness or {}`
2) Change the default for `self._staleness` to `{}` and update this method to return `None` if `self._staleness is {}`

Which one we use depends on whether getting `None` or an empty dict is more useful/clearer when calling `conn.staleness`